manifests/windows/install.pp

```diff
@@ -45,15 +44,15 @@
   }
 
   $_timestamp = strftime('%Y_%m_%d-%H_%M')
-  $_logfile = "${::env_temp_variable}\\puppet-${_timestamp}-installer.log"
+  $_logfile = windows_native_path("${::env_temp_variable}/puppet-${_timestamp}-installer.log")
   notice ("Puppet upgrade log file at ${_logfile}")
   debug ("Installing puppet from ${_msi_location}")
-  file { "${::env_temp_variable}\\install_puppet.bat":
+  file { "${_installbat}":
     ensure  => file,
     content => template('puppet_agent/install_puppet.bat.erb')
   }->
   exec { 'install_puppet.bat':
-    command => "${::system32}\\cmd.exe /c start /b ${_cmd_location} /c \"${::env_temp_variable}\\install_puppet.bat\" ${::puppet_agent_pid}",
+    command => "${::system32}\\cmd.exe /c start /b ${_cmd_location} /c \"${_installbat}\" ${::puppet_agent_pid}",
```

Comment:
@Iristyle Are there any additional complications that come from installing the msi on top of an existing installation?  (In this case a puppet-agent msi replacing a pe-puppet/puppet 3.x msi?).  What we're seeing is that the puppet.conf is being overwritten, so the server setting is lost (I think).